src/stylesheets/App.css

```diff
@@ -0,0 +1,355 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&family=Roboto+Slab:wght@100;200;300;400;500;600;700&display=swap');
+
+:root {
+  --azure: #0290ff;
+  --gray: #e8e8e8;
+  --black: #121212;
+  --blue: #4386bf;
+  --medium-blue: #307bbe;
+  --dodger-blue: #379cf6;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fafafa;
+  font-family: 'Montserrat', sans-serif;
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+li {
+  list-style: none;
+}
+
+.flex {
+  display: flex;
+}
+
+.p-100 {
+  padding: 0 100px;
+}
+
+.align-c {
+  align-items: center;
+}
+
+.just-c {
+  justify-content: center;
+}
+
+.just-sb {
+  justify-content: space-between;
+}
+
+.pos-rel {
+  position: relative;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+.op-5 {
+  opacity: 0.5;
+}
+
+.fw-300 {
+  font-weight: 300;
+}
+
+.nav {
+  padding: 30px;
+  background-color: #fff;
+  border-bottom: 2px solid var(--gray);
+  margin-bottom: calc(38px - 15px);
+}
+
+.title {
+  color: var(--azure);
+  font-size: 30px;
+  font-weight: bolder;
+  text-decoration: none;
+  margin-left: 0;
+}
+
+.right-cont {
+  margin-left: 50px;
+}
+
+.book-header-title {
+  margin-right: 40px;
+  text-transform: uppercase;
+  font-size: 15px;
+  letter-spacing: 1.9px;
+  color: var(--black);
+  margin-top: 15px;
+}
+
+#filterCategories {
+  text-transform: uppercase;
+  color: gray;
+  font-size: 15px;
+  margin-right: 10px;
+  letter-spacing: 1.9px;
+  border-color: white;
+}
+
+.first-container {
+  width: 20%;
+}
+
+.second-container {
+  display: flex;
+  flex-direction: row;
+  width: 80%;
+}
+
+.prof {
+  width: 50px;
+  height: 50px;
+  border: solid 1px var(--gray);
+  border-radius: 50%;
+  position: absolute;
+  right: 100px;
+  bottom: 23px;
+}
+
+.prof svg {
+  font-size: 1.25em;
+}
+
+.prof svg path {
+  fill: var(--azure);
+}
+
+.book-card {
+  font-family: 'Roboto Slab', sans-serif;
+  margin: 15px 0;
+  padding: 32px 147px 26px 27px;
+  border: solid 1px var(--gray);
+  border-radius: 4px;
+  position: relative;
```

Comment:
Please kindly use the Design background color  for each of your `.book-card` container class current the white background color white is not showing but I think it was working initially kindly fix it
![Screenshot 2021-09-18 153012](https://user-images.githubusercontent.com/38242590/133892451-a746ef2c-b194-43b2-b190-d6a59979c207.png)